flaskbb/plugins/spec.py

```diff
@@ -197,6 +197,7 @@ def flaskbb_event_topic_save_after(topic, is_new):
     """
 
 
+# TODO(anr): When pluggy 1.0 is released, mark this spec deprecated
```

Comment:
I think pluggy 1.0 will deprecate the ``implprefix`` prefix as well - that would mean that we have to declare all hooks explicitly right?

Reply:
from my understanding, implicitly gathering non-spec decorated hooks is going away. since we decorate ours, we should be fine

Reply:
ahh ok. I didn't know that this was possible.